blobstore/src/test/java/org/jclouds/blobstore/integration/internal/BaseBlobIntegrationTest.java

```diff
@@ -432,7 +433,13 @@ public void deleteObject(String name) throws InterruptedException {
       String container = getContainerName();
       try {
          addBlobToContainer(container, name, name, MediaType.TEXT_PLAIN);
+         if (view.getConsistencyModel() == ConsistencyModel.EVENTUAL) {
```

Comment:
It would be great to replace these kinds of comparisons with `Assert.isEqualTo(Object expected)` available to us in assertj. Just an idea, WDYT?